boot.py

```diff
@@ -1,30 +1,42 @@
-from .plugin.core.main import startup, shutdown
-
-# TODO: narrow down imports
-from .plugin.core.panels import *
+# Please keep this list sorted (Edit -> Sort Lines)
+from .plugin.code_actions import LspCodeActionBulbListener
+from .plugin.code_actions import LspCodeActionsCommand
```

Comment:
I assume 

```py
from .plugin.code_actions import LspCodeActionBulbListener, LspCodeActionsCommand
```

is preferred in this github organization?

Reply:
It doesn't really matter that much. But I prefer having each import on a separate line and alphabetically sorted. This makes diffs easier to review, and makes merge conflicts much easier to resolve. If I'm adding a new import and the other imports have a certain style, I'll try to keep to that style. It depends on the context.